src/classes/AWSUtilityTest.cls

```diff
@@ -1,21 +1,27 @@
-@isTest
+@isTest(SeeAllData=true)
```

Comment:
SeeAllData shouldn't be used in test unless it's absolutely necessary as it is a bad practice.
It doesn't look like this test method depends on it anyway